package.json

```diff
@@ -16,7 +16,7 @@
     "firebase-admin": "^12.1.1"
   },
   "scripts": {
-    "install": "npm --prefix functions/models install --save-dev && npm --prefix functions install --save-dev",
```

Comment:
Could this have simply caused the need for specifying typescript as an actual dependency rather than a dev one?